post-mortems-upgrades/upgrade-instructions/shockwave-omega.md

```diff
@@ -8,6 +8,24 @@
 - Make sure to [backup your validator](../../node-runners/validator-backup.md) before making any chnages.
 - **Please read carefully before you begin the upgrade.**
 
+## Check your hardware compatiblity before the upgrade
+
+To check whether your hardware will be compatible with the new release, you can run the `check-hw` tool of the new release (or the latest RC version, if not available). You can find it in the [Releases page](https://github.com/scrtlabs/SecretNetwork/releases).
```

Comment:
To check whether your hardware will be compatible with the new release -> To check whether your hardware will be able to register on the network*